pom.xml

```diff
@@ -154,18 +176,18 @@
             <artifactId>junit</artifactId>
             <version>4.13.2</version>
             <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.hamcrest</groupId>
+                    <artifactId>hamcrest-core</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
-          <groupId>org.hamcrest</groupId>
-          <artifactId>hamcrest</artifactId>
-          <version>2.2</version>
-          <scope>test</scope>
-        </dependency>
-        <dependency>
-          <groupId>org.hamcrest</groupId>
-          <artifactId>hamcrest-core</artifactId>
-          <version>2.2</version>
-          <scope>test</scope>
+            <groupId>org.hamcrest</groupId>
```

Comment:
Why did these 4 lines change? White space differences? If so, can you eliminate those differences?

Reply:
The pom itself conflicts all over with its formatting style (2 space or 4 space).  The code immediately above and below are using 4 so I adjusted it to stick to 4.

Reply:
On review, there should be a ignore whitespace that will help not see the issue.  It might also be advisable to format the pom overall separately so it adheres to 2 or 4 space setting.  Using sortpom-maven-plugin is great tool to assist in that as it is compliant to maven standards but would likely hit a lot at once.  That might be something to address when deviation of main to 1.6.5 are finalized and right before release (provided not a lot of other branches that might affect on the pom with that larger formatting effort).

Reply:
I actually plan to add a style formatter to the entire project which will auto format the code, xml files, etc. I plan to do that after doing this release. I already did this for the OWASP BenchmarkJava project, which I also maintain.